bitwarden_license/src/app/organizations/manage/sso.component.html

```diff
@@ -32,9 +32,14 @@ <h1>{{'singleSignOn' | i18n}}</h1>
             </label>
         </div>
         <div class="form-check mt-2 form-check-block">
-            <input class="form-check-input" type="radio" id="memberDecryptionKey" [value]="true" formControlName="useKeyConnector">
+            <input class="form-check-input" type="radio" id="memberDecryptionKey" [value]="true" formControlName="useKeyConnector"
+                [attr.disabled]="organization.useKeyConnector || null">
```

Comment:
You said you did this because you only want to disable a single radio button. But what is the point of disabling a single radio button when there are only 2 options? It seems that we could just disable the entire FormControl and the UX would be no different, and it would be a lot cleaner.

Reply:
We decided to leave it as is.